content/docs/start/data-pipelines.md

```diff
@@ -93,18 +93,18 @@ prepare:
   two files in it. This is how the <abbr>workspace</abbr> should look like now:
 
   ```diff
-      .
-      ├── data
-      │   ├── data.xml
-      │   ├── data.xml.dvc
+        .
+        ├── data
+        │   ├── data.xml
+        │   ├── data.xml.dvc
   +   │   └── prepared
   +   │       ├── test.tsv
```

Comment:
Hmmm I don't know. Yeah, this aligns the `|` characters but there's several other diff examples here and there and they all have indentation problems when rendered.

Reply:
It also makes it extra obvious which lines are being added the way it is now.

Reply:
@jorgeorpinel `+` make that obvious, but the visual layout looks outright broken. Maybe the issue here is that we are using `diff` format, while this is not actually a patch. We could higlight new entries some other way, e.g. by making it bald  or by adding a bash-like comment suffix that looks smth like  `# added` or something.

Reply:
This patch keeps `+` to make added entries obvious and also fixes the layout. :+1:

Reply:
> the visual layout looks outright broken
Agree, but we have an issue to fix this 🙂 #1761

> using diff format, while this is not actually a patch
So `git diff` doesn't output proper diff format? Or is something missing in this snippet to make it valid?

> We could higlight new entries some other way
Sure, that's a good idea. IMO this can be left alone though (since the bug should be addressed eventually).

> making it bald
👨‍🦲

Reply:
@jorgeorpinel how would that output look in git diff? How would it look on the GH? What about if we copy/paste it from git diff or GH - will we have extra spaces?

Reply:
Let me check... ⌛

Reply:
Oh sorry I was confused, this has nothing to do with `git diff`, I think it's just supposed to be a regular `diff` of outputs of `tree`..? Not sure, you created this one Ivan 🙂

Unfortunately, `tree` on Windows uses different characters and it doesn't look good the way I tried it but please copy/paste the commands to double check:

```console
λ mkdir test && cd test
λ mkdir data/raw
λ touch data/raw/words.txt
λ touch data/raw/names.csv
λ tree > b
λ mkdir data/clean
λ touch data/clean/words.txt
λ tree > 2
λ diff b a
2a3
> |-- 2
4c5,7
<                     `-- raw
---
>     |               `-- clean
>     |               `-- words.txt
>     `-- raw
8c11
< 2 directories, 3 files
---
> 3 directories, 5 files
```

Reply:
Let me check what `git diff` does though... ⌛

Reply:
OK `git diff` would show something like this:

```console
λ mkdir -p data/raw
λ touch data/raw/words.txt
λ touch data/raw/names.csv
λ tree > tree
λ git add tree
λ mkdir data/clean
λ touch data/clean/words.txt
λ git diff
diff --git a/tree b/tree
index e60fc51..c90b158 100644
--- a/tree
+++ b/tree
@@ -1,8 +1,10 @@
 .
 |-- data
 |
+|               `-- clean
+|   |               `-- words.txt
+|
    `-- raw
 |       |           `-- names.csv
 |                   `-- words.txt
 `-- tree

-2 directories,
 3
+directories, 4
  files
```

OK so never mind, I was confused again... I guess the intention here was to output whatever format `git diff` does, and the conclusions from above are:

1. `git diff` does align stuff adding **one space** to lines without `+`
2. Indeed this is not `diff` format. We should probably not use that language name in any of the existing blocks throughout docs...

Reply:
>  We should probably not use that language name in any of the existing blocks throughout docs...

is there some other prisma.js lang that we should use that would highlight it similar to the GH interface?

Reply:
Good Q. Idk. Do we know what format does `git diff` use BTW? I'll try some options from https://prismjs.com/#supported-languages ⏳

Reply:
p.s. OK `diff` uses diff format, while `git diff` seems to use "patch" format. Prism doesn't have a `patch` language, BUT it has a `git` language, which looks pretty good for this 🙂 — will update here.

> Leaving this thread unresolved for future ref.